packages/sales-api-service/src/services/renewals/__tests__/renewals.service.spec.js

```diff
@@ -107,7 +136,42 @@ describe('preparePermissionDataForRenewal', () => {
         licenceStartTime: endDate.hours(),
         renewedEndDate: endDate.toISOString()
       }
-      expect(preparePermissionDataForRenewal(existingPermission({ endDate }))).toEqual(expect.objectContaining(expectedData))
+      expect(await preparePermissionDataForRenewal(existingPermission({ endDate }))).toEqual(expect.objectContaining(expectedData))
+    })
+  })
+
+  describe('prepareConcessionsData', () => {
```

Comment:
Could we also add some tests to make sure that disability concessions are also handled appropriately?